src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -369,6 +369,89 @@ def translate_literal(
     return [(data_node, data_type)]
 
 
+def translate_scalar_expr(
+    node: gtir.Node,
+    sdfg: dace.SDFG,
+    state: dace.SDFGState,
+    sdfg_builder: gtir_to_sdfg.SDFGBuilder,
+    reduce_identity: Optional[gtir_to_tasklet.SymbolExpr],
+) -> list[TemporaryData]:
+    assert isinstance(node, gtir.FunCall)
+    assert isinstance(node.type, ts.ScalarType)
+
+    args = []
+    connectors = []
+    scalar_expr_args = []
+
+    for arg_expr in node.args:
+        is_symbol = True
+        if isinstance(arg_expr, gtir.SymRef):
+            try:
+                sdfg_builder.get_symbol_type(arg_expr.id)
+            except KeyError:
+                is_symbol = False
+
+        if is_symbol:
```

Comment:
With symbol do you mean a DaCe symbol?

Reply:
No, I mean a symbol defined in the GT4Py program. Do you have any name suggestion how to distinguish them? It is probably confusing everywhere in this code. Maybe `gt_symbol` and `dc_symbol`?

Reply:
I agree. I like `gt_symbol` very much, but `dc_symbol` not so much, however, since I have no better idea, I would go along with it.